components/alerts/alerts.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
@jamesmadson I don't think we need this file in here anymore; however, if you think it'd be helpful to leave breadcrumbs for what a component `config` file could look like, should we generalize it or add additional comments? For example, I removed the `context` tab via the Fractal config so I don't think those lines would display anyway which should probably be noted. Thoughts?

Reply:
That would be good, yeah. Would you mind doing that?

Reply:
Sure, will do!